typings/tests/hook.tsx

```diff
@@ -2,14 +2,6 @@ import React from "react";
 import { useDropzone, DropzoneProps } from "../../";
 
 export const Dropzone = ({ children, ...opts }: DropzoneProps) => {
-  const { ...state } = useDropzone(opts);
-  return children(state);
+  const state = useDropzone(opts);
+  return <>{children(state)}</>;
 };
-
-<Dropzone>
-  {({ getRootProps, getInputProps }) => (
-    <div {...getRootProps()}>
-      <input {...getInputProps()} />
-    </div>
-  )}
-</Dropzone>;
```

Comment:
Why was this removed?

Reply:
I'm curious too

Reply:
because this code is useless

Reply:
I don't think this code makes any sense
because there are no exports and references

Reply:
I don't think we've ever used these codes
because there are no exports and import
<img width="1560" alt="Screen Shot 2022-04-27 at 11 05 45" src="https://user-images.githubusercontent.com/54422571/165432326-255f3ebc-d5b7-4b6f-97f7-688dce052852.png">

Reply:
The purpose of it was to check that using the hook syntax doesn't cause any TS errors. So please add it back.

Reply:
This is an optimization, I can delete it if you don't like it

Reply:
ok, i have reverted the code
But I still want to know why not use the following way:
`
 return <>{children(state)}</>;
`
In this way, ts will not report errors, and the syntax is more streamlined